### DISCUSSION TITLE
When using easy GA config, does it respect user opt-out?

### DISCUSSION THREAD

**Discussion by StarfallProjects** [squidfunk/mkdocs-material#4413] [Q&A] [answered]
Can I confirm whether, when using the easy GA setup (adding tracking code to mkdocs.yml), it respects the user choice to opt out? I can't see anything in the generated tracking code that checks for user options before sending data?

Edit to add: config is here https://github.com/n8n-io/n8n-docs/blob/restore-GA/mkdocs.yml

**Comment by squidfunk** [comment, marked as answer]
That's exactly what the [cookie consent](https://squidfunk.github.io/mkdocs-material/setup/ensuring-data-privacy/#cookie-consent) is for. If you enable it, the user will be asked _prior_ to sending requests to Google Analytics (= opt-in) and can change settings after-wards. She can also [opt-out again afterward](https://squidfunk.github.io/mkdocs-material/setup/ensuring-data-privacy/#change-cookie-settings).

**Reply by StarfallProjects** [reply]
Cheers. I'd enabled it, and tried opting out, but it wasn't clear that it wasn't sending info. Thanks for confirming!
